packages/go_router/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: go_router
 description: A declarative router for Flutter based on Navigation 2 supporting
   deep linking, data-driven routes and more
-version: 5.1.1
+version: 5.1.1+1
```

Comment:
I think this change deserve a version bump

Reply:
patch should be fine.

Reply:
Done